### PR TITLE
Wait for peers connection.

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**
FIx waiting for peers connection

### CODE REVIEW

File: itests/node_client/node_client.go [hunk 1]
```diff
@@ -101,3 +101,21 @@ func (c *NodesClients) ClearBlackList(t *testing.T) {
 	c.GoClients.HttpClient.ClearBlackList(t)
 	c.ScalaClients.HttpClient.ClearBlackList(t)
 }
+
+func (c *NodesClients) WaitForConnectedPeers(t *testing.T, timeout time.Duration) (error, error) {
+	errGo := Retry(timeout, func() error {
+		cp, _, err := c.GoClients.HttpClient.ConnectedPeers(t)
+		if len(cp) == 0 && err == nil {
+			err = fmt.Errorf("no connected peers")
```
Comment (alexeykiselev) [inline itests/node_client/node_client.go:109]: Please use `errors.New()` here. Try not to use `fmt.Errorf`

File: itests/node_client/node_client.go [hunk 2]
```diff
@@ -101,3 +101,21 @@ func (c *NodesClients) ClearBlackList(t *testing.T) {
...
+		}
+		return err
+	})
+	errScala := Retry(timeout, func() error {
+		cp, _, err := c.ScalaClients.HttpClient.ConnectedPeers(t)
+		if len(cp) == 0 && err == nil {
+			err = fmt.Errorf("no connected peers")
```
Comment (alexeykiselev) [inline itests/node_client/node_client.go:116]: The same as above

**Review by alexeykiselev — CHANGES_REQUESTED** [review]

File: itests/node_client/node_client.go (hunk 1 above)
Comment (irina-pereiaslavskaia) [reply itests/node_client/node_client.go]: Done

File: itests/node_client/node_client.go (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply itests/node_client/node_client.go]: Done

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**